plugins/gradle/java/src/execution/GradleRunnerUtil.java

```diff
@@ -112,6 +112,9 @@ public static Couple<String> parseComparisonMessage(String exceptionMsg) {
     if (comparisonPair == null) {
       comparisonPair = parseComparisonMessage(exceptionMsg, "\nExpected: \"(.*)\"\n\\s*but: was \"(.*)\"");
     }
+    if (comparisonPair == null) {
+      comparisonPair = parseComparisonMessage(exceptionMsg, "\nexpected: \"(.*)\"\n\\s*but was: \"(.*)\"");
```

Comment:
Would you update the previous regexp instead so it covers both cases?

Reply:
I could use a regex like

```
\n[Ee]xpected: "(.*)"\n\s*but:? was:? "(.*)"
```

but it wouldn't be semantically the same as this code, because there's a disconnect between the `E` vs `e` and the `but: was` vs `but was:`. For example, this regex would erroneously match the string

`
Expected: "foo"
but was: "Foo"
`

whereas the code (as written) would not.

Reply:
Well, that is fair, however, does it make any difference in practice?
Frankly speaking, it is demotivating that IDE has to execute the same regular expressions several times, especially when the two do not differ much.

Reply:
I'm not aware of a case, such as the one I provided earlier, that is practical. I combined the regexs.